### ISSUE TITLE
Language selection not working

### ISSUE THREAD

**Issue by raspyweather** [WikiJourney/wikijourney_website#68]
After selecting a language, the side redirects from
[https://wikijourney.eu/action.php?lg=en](https://wikijourney.eu/action.php?lg=en)
instantly to
[https://wikijourney.eu/action.php](https://wikijourney.eu/action.php)
without changing the language at all.

After some investigation, it seems that the lg cookie gets not set correctly. It has the initial value  `fr`, changing the language does not change it. By manipulating it manually to `en` the site works in english.

I'm not sure if that's part of the problem, but your `piwik.js` script gets not loaded properly because of a certificate problem. By ignoring the certificate problem or trying over http, I get a `Site not found` error.
[https://piwik.wikijourney.eu/piwik.js]( https://piwik.wikijourney.eu/piwik.js)

 Tested with:
- Chrome 59.0.3071.109 
- Firefox 56

**Comment by sylvainar** [comment]
Yup this is actually a problem. We made a server migration not so long ago and we missed this bug. I'm gonna investigate soon. Thanks for reporting !

**Comment by raspyweather** [comment]
Any update so far?
